packages/financial-templates-lib/test/clients/FinancialContractEventClient.js

```diff
@@ -63,6 +63,11 @@ const versionedIt = function (supportedVersions, shouldBeItOnly = false) {
   return runTestForVersion(supportedVersions, TESTED_CONTRACT_VERSIONS, iterationTestVersion) ? it : () => {};
 };
 
+const objectsInArrayInclude = (subset, superset) => {
+  assert.equal(superset.length, subset.length);
+  for (let i = 0; i < superset.length; i++) assert.deepInclude(superset[i], subset[i]);
```

Comment:
i feel like one of these should not be indexed if you are using deepInclude, probably the one with the array of possible values

Reply:
Hmm, what do you mean. The expectation is that you have an array of subsets and supersets and each subsequent subset should have each of its values in the corresponding superset.

I may be missing something, though.

Reply:
yea i misunderstood what was happening here, after you explained it, and looking at chai docs again it makes sense.